qa/src/main/java/com/jkh/FE/ConfigurationFE.java

```diff
@@ -0,0 +1,13 @@
+package com.jkh.FE;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ConfigurationFE {
+
+    @Value("${currentBrowser:firefox}")
```

Comment:
We will use firefox as the main browser? should it be google chrome? or it doesn't make sense?

Reply:
it doesn't make sense